### ISSUE TITLE
DROP CLUSTER isn't working

### ISSUE THREAD

**Issue by vsspt** [orientechnologies/orientdb#7807]
## Expected behavior  
When executing a DROP CLUSTER instruction, the data should be removed, and the CLUSTER physical files removed.

## Actual behavior  
After executing a DROP CLUSTER instruction the data is not removed, and the  CLUSTER physical files still exist.
NOTE: TRUNCATE CLUSTER works as expected.

## Steps to reproduce  
`CREATE CLASS `TABLE_TEST`

CREATE CLUSTER test_1
ALTER CLASS `TABLE_TEST` ADDCLUSTER test_1

INSERT INTO TABLE_TEST CLUSTER test_1 (ID) VALUES ('1');
SELECT COUNT(*) from TABLE_TEST
> 1

DROP CLUSTER test_1
SELECT COUNT(*) from TABLE_TEST
> 1

TRUNCATE CLUSTER test_1
SELECT COUNT(*) from TABLE_TEST
> 0
`

**Comment by luigidellaquila** [comment]
Hi @vsspt 

Thank you for reporting, I'll check it asap

Luigi

**Comment by luigidellaquila** [comment]
Hi @vsspt 

If you see the output of DROP CLUSTER query, you will see that it returns `false`, that means that the cluster cannot be removed.

This happens because the cluster is in use for one of the classes (TABLE_TEST in this case).
To drop it you have to:

```
alter class TABLE_TEST REMOVECLUSTER test_x;
DROP CLUSTER test_1
```

I'll anticipate your objection: it's impossible here to understand what's going on, there is no info in the result. I just wanted to let you know that it's already fixed in develop branch, so v 3.0 will have a much clearer result.

Thanks

Luigi

**Comment by vsspt** [comment]
@luigidellaquila ,

i executed the above code, but i'm getting an error in the ALTER CLASS statement:

`alter class TABLE_TEST REMOVECLUSTER test_1;`

`com.orientechnologies.orient.core.exception.OCommandExecutionException: Error on execution of command: sql.alter class TABLE_TEST REMOVECLUSTER test_1
	DB name="testDB"
	at com.orientechnologies.orient.core.storage.impl.local.OAbstractPaginatedStorage.executeCommand(OAbstractPaginatedStorage.java:3361)
	at com.orientechnologies.orient.core.storage.impl.local.OAbstractPaginatedStorage.command(OAbstractPaginatedStorage.java:3278)
	at com.orientechnologies.orient.core.command.OCommandRequestTextAbstract.execute(OCommandRequestTextAbstract.java:69)
	at com.orientechnologies.orient.server.network.protocol.http.command.post.OServerCommandPostCommand.execute(OServerCommandPostCommand.java:106)
	at com.orientechnologies.orient.graph.server.command.OServerCommandPostCommandGraph.execute(OServerCommandPostCommandGraph.java:37)
	at com.orientechnologies.orient.server.network.protocol.http.ONetworkProtocolHttpAbstract.service(ONetworkProtocolHttpAbstract.java:173)
	at com.orientechnologies.orient.server.network.protocol.http.ONetworkProtocolHttpAbstract.execute(ONetworkProtocolHttpAbstract.java:629)
	at com.orientechnologies.common.thread.OSoftThread.run(OSoftThread.java:81)
Caused by: java.lang.IllegalArgumentException: Cluster id 'test_1' cannot be removed
	at com.orientechnologies.orient.core.metadata.schema.OClassImpl.set(OClassImpl.java:1697)
	at com.orientechnologies.orient.core.sql.OCommandExecutorSQLAlterClass.execute(OCommandExecutorSQLAlterClass.java:173)
	at com.orientechnologies.orient.core.sql.OCommandExecutorSQLDelegate.execute(OCommandExecutorSQLDelegate.java:70)
	at com.orientechnologies.orient.core.storage.impl.local.OAbstractPaginatedStorage.executeCommand(OAbstractPaginatedStorage.java:3340)
	... 7 more
$ANSI{green {db=testDB}} Exception `3711363B` in storage `testDB`
com.orientechnologies.orient.core.exception.OCommandExecutionException: Error on execution of command: sql.alter class TABLE_TEST REMOVECLUSTER test_1
	DB name="testDB"
	at com.orientechnologies.orient.core.storage.impl.local.OAbstractPaginatedStorage.executeCommand(OAbstractPaginatedStorage.java:3361)
	at com.orientechnologies.orient.core.storage.impl.local.OAbstractPaginatedStorage.command(OAbstractPaginatedStorage.java:3278)
	at com.orientechnologies.orient.core.command.OCommandRequestTextAbstract.execute(OCommandRequestTextAbstract.java:69)
	at com.orientechnologies.orient.server.network.protocol.http.command.post.OServerCommandPostCommand.execute(OServerCommandPostCommand.java:106)
	at com.orientechnologies.orient.graph.server.command.OServerCommandPostCommandGraph.execute(OServerCommandPostCommandGraph.java:37)
	at com.orientechnologies.orient.server.network.protocol.http.ONetworkProtocolHttpAbstract.service(ONetworkProtocolHttpAbstract.java:173)
	at com.orientechnologies.orient.server.network.protocol.http.ONetworkProtocolHttpAbstract.execute(ONetworkProtocolHttpAbstract.java:629)
	at com.orientechnologies.common.thread.OSoftThread.run(OSoftThread.java:81)
Caused by: java.lang.IllegalArgumentException: Cluster id 'test_1' cannot be removed
	at com.orientechnologies.orient.core.metadata.schema.OClassImpl.set(OClassImpl.java:1697)
	at com.orientechnologies.orient.core.sql.OCommandExecutorSQLAlterClass.execute(OCommandExecutorSQLAlterClass.java:173)
	at com.orientechnologies.orient.core.sql.OCommandExecutorSQLDelegate.execute(OCommandExecutorSQLDelegate.java:70)
	at com.orientechnologies.orient.core.storage.impl.local.OAbstractPaginatedStorage.executeCommand(OAbstractPaginatedStorage.java:3340)
	... 7 more`
